community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/LogFileInformation.java

```diff
@@ -38,7 +38,7 @@
     /**
      * @return the last committed transaction id for this Log
      */
-    long getLastCommittedTxId();
+    long getLastCommittedId();
```

Comment:
There are other methods in this class `#getFirstExistingTxId()` and `#getFirstCommittedTxId(long)` and they still refer to transaction ids. This seems a bit inconsistent.